src/Functions/FunctionsCodingIP.cpp

```diff
@@ -580,7 +580,7 @@ class FunctionIPv4ToIPv6 : public IFunction
 #if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
             unalignedStoreLittleEndian<UInt64>(buf + 8, 0x00000000FFFF0000ull | (static_cast<UInt64>(ntohl(in)) << 32));
 #else
-            unalignedStoreLittleEndian<UInt64>(buf + 8, 0x00000000FFFF0000ull | (static_cast<UInt64>(__builtin_bswap32(in))) << 32));
```

Comment:
What was wrong with `__builtin_bswap32`?

Reply:
There wasn't anything wrong it, it's just `std::byteswap` is a more generic alternative starting from C++23.